configuration/db_mongo.rst

```diff
@@ -24,7 +24,7 @@ In this version issue `SERVER-7568 <https://jira.mongodb.org/browse/SERVER-7568>
 * If you have your own database in MongoDB already, change the ``MongoDbOptions`` to reflect your settings::
 
    "MongoDbOptions": {
-       "ConnectionString": "mongodb://localhost/vonkstu3",
+       "ConnectionString": "mongodb://localhost/vonkdata",
```

Comment:
When you are already changing the databasename, then this is the time to get rid of the name `vonk`, right?